src/docs/installing/installing-linux.md

```diff
@@ -14,19 +14,15 @@ Only Linux kernel 6.4 or newer is supported.
 ## Recommended Distributions
 
 ::: warning
-Ubuntu and Ubuntu-based distributions **may have issues** and are **not supported**.
-:::
-
-::: warning
-Debian versions older than Debian 12 (Bookworm) are **not supported**. Debian 12 (Bookworm) requires a custom kernel, the [audio script](#fixing-audio) will automatically install it for you.
+Old (>1 year) LTS releases **may have issues** and are not supported.  
+Debian 12 (Bookworm) and possibly Ubuntu require a custom kernel. In case of Debian, the [audio script](#fixing-audio) will automatically install it for you.
```

Comment:
Suggestion: One possible workaround for Debian 12 (Bookworm) and Ubuntu is using a custom kernel. In case of Debian, the [audio script](#fixing-audio) will automatically install it for you. Note that not all issues can be solved with a custom kernel, as the rest of the software on the system is still old.